files/plugins/cron_cis_audit.py

```diff
@@ -47,26 +48,36 @@ def _get_major_version():
 
 MAX_SLEEP = 600
 PID_FILENAME = "/tmp/cron_cis_audit.pid"
+TAILORING_CIS_FILE = Path("/etc/usg/default-tailoring.xml")
 
 
 def _get_cis_hardening_profile(profile):
-    """Try to read the cis profile from cloud init log or default to level1_server."""
+    """Try to read the cis profile from cloud init log or default to level1_server.
+
+    If exists the "default" tailoring file in /etc/usg/default-tailoring.xml,
+    no profile is passed.
+    """
     if profile in PROFILES:
-        return profile
+        return [profile]
```

Comment:
This returns a profile, ignoring whether the tailoring file exists.  Does this contradict the docstring?